content/articles/getting-started-with-react-strapi-graphql/index.md

```diff
@@ -0,0 +1,343 @@
+### Getting Started With React, Strapi, and GraphQL
+
+Strapi is a headless CMS(Content Management System) written in JavaScript that enables users to create APIs with ease through a user-friendly interface. In traditional Content Management Systems, the frontend and backend parts of a website are usually bundled together. With a headless CMS, the two parts are separated and we can customize our frontend the way we want and create it with the languages or frameworks that we prefer.
+
+In this article, we will take a look at how we can use the Strapi CMS and React to create a blog web application. You can read more about Strapi in the [official documentation](https://docs.strapi.io/developer-docs/latest/getting-started/quick-start.html).
+
```

Comment:
Please add a table of content here, linking them to their various heading for easy navigation to the reader.